### ISSUE TITLE
Kappa History

### ISSUE THREAD

**Issue by RatexIndex** [ztizzlegaming/CSGOWinBig#48]
@ztizzlegaming

We need a Legend for all Kappa History Issues please in a Pastebin Document can you add a Pastebin Doc pls who i have the rights with Anthony NL  than we add faq and all other sh1t ^^?

**Comment by TheAnthonyNL** [comment]
Ehm ... and what exactly is your question ~_~...

**Comment by RatexIndex** [comment]
Nothink add a Kappa History Anthony you know what i mean xD

**Comment by andrewda** [comment]
What? A "Kappa History"?

**Comment by RatexIndex** [comment]
All says Issues but only the most people cant read it right, so we need a history for noobs

**Comment by andrewda** [comment]
Alright, well a PasteBin doesn't make sense. Add to this repository's wiki!

**Comment by RatexIndex** [comment]
andrewa do you need to type all time in my posts or what is wrong :D ?

**Comment by andrewda** [comment]
@RatexIndex no, I was just saying that the CSGOWinBig wiki makes much more sense for this type of stuff (https://github.com/ztizzlegaming/CSGOWinBig/)

**Comment by RatexIndex** [comment]
i make no history for all do we have now all time github newbies ? that is the powerfull question ;)
